context/start_jupyter.sh

```diff
@@ -4,9 +4,15 @@
 if [[ "${JUPYTER_FG}" == "true" ]]; then
    jupyter-lab --allow-root --ip=0.0.0.0 --no-browser --NotebookApp.token=''
    exit 0
+else
+   set -x
+   nohup jupyter-lab --allow-root --ip=0.0.0.0 --no-browser --NotebookApp.token='' > /dev/null 2>&1 &
+   set +x
```

Comment:
```suggestion
   nohup jupyter-lab --allow-root --ip=0.0.0.0 --no-browser --NotebookApp.token='' > /dev/null 2>&1 &
```

might be better to just remove the `set -/+ x` here. looks like it clutters up the output message a bit. I'm not sure why it was ever included to be honest.

![image](https://user-images.githubusercontent.com/7400326/89923159-2d6bb200-dbce-11ea-92c5-829689847f22.png)

Reply:
Fine with me!